bin/couchbackup.bin.js

```diff
@@ -51,15 +51,24 @@ if (program.output) {
   ws = fs.createWriteStream(null, { fd: fd });
 }
 
+var firstWrite = true;
+console.error('Fetching all database changes...');
+
 return couchbackup.backup(
   databaseUrl,
   ws,
   opts,
   error.terminationCallback
-).on('written', function(obj) {
-  debug('written', obj.batch, ' docs: ', obj.total, 'Time', obj.time);
+).on('changes', function(batch) {
+  process.stderr.write(`\r  Total batches received: ${batch + 1}`);
```

Comment:
I'm not sure if it is worth over-writing this line, we don't overwrite when receiving the written events. I know you have just kept things in line with what was here previously, but I'm not sure it is necessary. Why not just replace this and the `firstWrite` stuff with a `debug` statement for each change line?